packages/router/src/grpc/custody/authorize.ts

```diff
@@ -15,6 +18,19 @@ export const authorize: Impl['authorize'] = async (req, ctx) => {
   const sess = ctx.values.get(extSessionCtx);
   const local = ctx.values.get(extLocalCtx);
 
+  const services = ctx.values.get(servicesCtx);
+  const walletServices = await services.getWalletServices();
+  const assetsMetadata = await walletServices.indexedDb.getAllAssetsMetadata();
+  const denomMetadataByAssetId = assetsMetadata.reduce<Record<string, DenomMetadataAsJson>>(
+    (prev, curr) => {
+      if (curr.penumbraAssetId) {
+        prev[uint8ArrayToBase64(curr.penumbraAssetId.inner)] = curr.toJson();
```

Comment:
i think it might be better to use bech32 like in discussion, bech32 is the canonical string representation of an assetid

Reply:
Ah OK, done!